renderdoc/driver/vulkan/vk_core.h

```diff
@@ -1579,6 +1585,11 @@ class WrappedVulkan : public IFrameCapturer
                                 VkPhysicalDevice physicalDevice, VkSurfaceKHR surface,
                                 uint32_t *pPresentModeCount, VkPresentModeKHR *pPresentModes);
 
+  IMPLEMENT_FUNCTION_SERIALISED(VkResult, vkGetPhysicalDeviceSurfacePresentModes2EXT,
+                                VkPhysicalDevice physicalDevice,
+                                const VkPhysicalDeviceSurfaceInfo2KHR *pSurfaceInfo,
+                                uint32_t *pPresentModeCount, VkPresentModeKHR *pPresentModes);
```

Comment:
This implementation of `VK_EXT_full_screen_exclusive` seems incomplete - the functions `vkGetDeviceGroupSurfacePresentModes2EXT`, `vkAcquireFullScreenExclusiveModeEXT` and `vkReleaseFullScreenExclusiveModeEXT` are all missing. They will need a similarly unwrap-and-pass-through implementation.